GAD 6 - Verbessertes Sortieren/tests/simplesort/DualPivotQuicksortTester.java

```diff
@@ -90,9 +89,9 @@ void printLine() {
             );
         }
 
-        void randomTestFactory (DualPivotFinder pivotFinder, int selectSortSize, int seed, int iterations,
-                                int maxLength, int minLength, int numbers) {
-            DualPivotQuicksort quicksort = new DualPivotQuicksort(pivotFinder, selectSortSize);
+        void randomTestFactory(DualPivotFinder pivotFinderSort, DualPivotFinder test, int selectSortSize, int seed, int iterations,
+                               int maxLength, int minLength, int numbers) {
+            DualPivotQuicksort quicksort = new DualPivotQuicksort(pivotFinderSort, selectSortSize);
```

Comment:
I can't see the benefit of this additional argument here, yet. All method invocations use the same values for both parameters. Could you please further describe this to me?

Reply:
Yeah, expect RandomPivot, because it's not returning the same value ever time its called, I can't use the same pivot finder which is used in the algorithm.